src/fparser/two/Fortran2003.py

```diff
@@ -252,7 +252,7 @@ class Program(BlockBase):  # R201
     use_names = ["Program_Unit"]
 
     @show_result
-    def __new__(cls, string):
+    def __new__(cls, string, _deepcopy=False):
         """Wrapper around base class __new__ to catch an internal NoMatchError
         exception and raise it as an external FortranSyntaxError exception.
 
```

Comment:
Please document the new `_deepcopy` argument.

Reply:
OK.